### PR TITLE
Replace secret key with pfx certificate for Jwt token (#1797)

### LINKED ISSUE #1797: Using a `.pfx` file for token generation in the TodoTemplate project
Currently, a constant string is used as the secret key to generate tokens in the template project. It is required to enhance this process by using a `.pfx` file.

### PR DESCRIPTION
**Pull request by milad-karimifard**
This closes #1797

### CODE REVIEW

**Review by ysmoradi — APPROVED** [review]

**Comment by ysmoradi** [comment]
Could you please test it on WSL too? @milad-karimifard

File: src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/TodoTemplate.Api.csproj [hunk 1]
```diff
@@ -77,4 +77,10 @@
         </EmbeddedResource>
     </ItemGroup>
 
+    <ItemGroup>
+      <None Update="BitJwtCertificate.pfx">
```
Comment (ysmoradi) [inline src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/TodoTemplate.Api.csproj:81]: Rename it to IdentityServerCertificate.pfx please

**Review by ysmoradi — CHANGES_REQUESTED** [review]

**Comment by ysmoradi** [comment]
Does it work with a pfx that is created by the following scripts?

```powershell
$cert = New-SelfSignedCertificate -CertStoreLocation Cert:\LocalMachine\My -Subject "IdentityServerCN" -Provider "Microsoft Strong Cryptographic Provider" -HashAlgorithm "SHA512" -NotAfter (Get-Date).AddYears(5)

Export-PfxCertificate -cert ('Cert:\LocalMachine\My\' + $cert.thumbprint) -FilePath IdentityServerCertificate.pfx -Password (ConvertTo-SecureString -String "P@ssw0rdP@ssw0rd" -Force -AsPlainText)
```

**Review by ysmoradi — APPROVED** [review]

File: src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/TodoTemplate.Api.csproj [hunk 2]
```diff
@@ -6,6 +6,7 @@
         <MvcRazorExcludeRefAssembliesFromPublish>false</MvcRazorExcludeRefAssembliesFromPublish>
         <PreserveCompilationReferences>true</PreserveCompilationReferences>
         <PreserveCompilationContext>true</PreserveCompilationContext>
+        <UserSecretsId>df8517d0-6b4c-46e5-a8b6-fe97ed6b0e96</UserSecretsId>
```
Comment (msynk) [inline src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/TodoTemplate.Api.csproj:9]: this line seems redundant, shouldn't it be removed? or did you intend to use the secret manager somehow?

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/TodoTemplate.Api.csproj (hunk 2 above)
Comment (ysmoradi) [reply src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/TodoTemplate.Api.csproj:9]: Remove it please @milad-karimifard

**Review by ysmoradi — APPROVED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**